### PR TITLE
Add PoliCheck

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Start compliance stage in the aggregate report pipeline.
- Add PoliCheck tool to the compliance stage

### CODE REVIEW

File: sdk/core/azure-core/azure/core/pipeline/policies/_universal.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ class HttpLoggingPolicy(SansIOHTTPPolicy):
     """The Pipeline policy that handles logging of HTTP requests and responses.
     """
 
-    DEFAULT_HEADERS_WHITELIST = set([
+    DEFAULT_HEADERS_ALLOWLIST = set([
```
Comment (weshaggard) [inline sdk/core/azure-core/azure/core/pipeline/policies/_universal.py:353]: @xiangyan99 is this going to cause any issues if we change the name here?

File: sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py [hunk 2]
```diff
@@ -45,14 +45,14 @@ class RedirectPolicyBase(object):
 
     REDIRECT_STATUSES = frozenset([300, 301, 302, 303, 307, 308])
 
-    REDIRECT_HEADERS_BLACKLIST = frozenset(['Authorization'])
+    REDIRECT_HEADERS_BLOCKLIST = frozenset(['Authorization'])
```
Comment (weshaggard) [inline sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py:48]: @xiangyan99 is this going to cause any issues if we change the name here?

File: sdk/core/azure-core/azure/core/pipeline/policies/_universal.py (hunk 1 above)
Comment (xiangyan99) [reply sdk/core/azure-core/azure/core/pipeline/policies/_universal.py:353]: This is a breaking change.

Given it is undocumented, I think it is acceptable.  But we may need to describe it in our changelog.

Your ideas? @johanste

Comment (johanste) [reply sdk/core/azure-core/azure/core/pipeline/policies/_universal.py:359]: I think this is acceptable (from a breaking change perspective). And the right thing to do.

File: sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py (hunk 2 above)
Comment (johanste) [reply sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py:48]: I think this is acceptable (from a breaking change perspective). And the right thing to do.

File: sdk/core/azure-core/azure/core/pipeline/policies/_universal.py (hunk 1 above)
Comment (weshaggard) [reply sdk/core/azure-core/azure/core/pipeline/policies/_universal.py]: @chidozieononiwu why did you remove this and the other instance from your changes?

Comment (chidozieononiwu) [reply sdk/core/azure-core/azure/core/pipeline/policies/_universal.py]: It turned out that some of the associated test is somehow requirign an already published package. After looking over it with @scbedd we to just ignore this for now.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**